src/App.jsx

```diff
@@ -1,13 +1,145 @@
+import FruitOverview from "./components/FruitOverview.jsx";
+import {useState} from "react";
+import FormField from "./components/FormField.jsx";
+
 
-import './styles/App.css'
 
 function App() {
+    // UseStates
+    const [strawberryCount, setStrawberryCount] = useState(0);
+    const [bananaCount, setBananaCount] = useState(0);
+    const [appleCount, setAppleCount] = useState(0);
+    const[kiwiCount, setKiwiCount] = useState(0);
+
+    const dropdownOptions = [{text : "iedere week", value : "weekly"}, {text: "om de week", value : "biweekly"}, {text : "iedere maand", value : "monthly"}];
+    const radioButtonOptions = [{text : "Overdag", value : "dag"}, { text : "in de avond", value: "avond"}];
+
+    const changeStrawberryCount = (operator) => {
+        if(operator === "remove" && strawberryCount === 0) return;
+        operator === "add" ? setStrawberryCount(strawberryCount + 1) : setStrawberryCount(strawberryCount - 1);
+    }
+
+    const changeBananaCount = (operator) => {
+        if(operator === "remove" && strawberryCount === 0) return;
+        operator === "add" ? setBananaCount(bananaCount + 1) : setBananaCount(strawberryCount - 1);
+    }
+
+    const changeAppleCount = (operator) => {
+        if(operator === "remove" && strawberryCount === 0) return;
+        operator === "add" ? setAppleCount(appleCount + 1) : setAppleCount(appleCount - 1);
+    }
+
+    const changeKiwiCount = (operator) => {
+        if(operator === "remove" && strawberryCount === 0) return;
+        operator === "add" ? setKiwiCount(kiwiCount + 1) : setKiwiCount(kiwiCount - 1);
+    }
+    const resetCounts = () => {
+        setStrawberryCount(0);
+        setBananaCount(0);
+        setAppleCount(0);
+        setKiwiCount(0);
+
+    }
+
+    function printFormInput() {
+        console.log("Good so far~`");
+    }
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+            <FruitOverview
+                imgSrc={strawberryIcon}
+                imgAlt="aarbei"
+                text="Aarbeien"
+                itemCount={strawberryCount}
+                btnFunction={changeStrawberryCount}
+            />
+            <FruitOverview
+                imgSrc={bananaIcon}
+                imgAlt="banaan"
+                text="Bananan"
+                itemCount={bananaCount}
+                btnFunction={changeBananaCount}
+            />
+            <FruitOverview
+                imgSrc={appleIcon}
+                imgAlt="appel"
+                text="appels"
+                itemCount={appleCount}
+                btnFunction={changeAppleCount}
+            />
+            <FruitOverview
+                imgSrc={kiwiIcon}
+                imgAlt="kiwi"
+                text="Kiwi's"
+                itemCount={kiwiCount}
+                btnFunction={changeKiwiCount}
+            />
+            <button onClick={resetCounts}>Reset</button>
```

Comment:
Ik zie best vaak een button terug komen, Wellicht is dat een kandidaat voor een component!